Products/PluggableAuthService/events.py

```diff
@@ -38,6 +42,22 @@ def __init__(self, principal):
         self.object = principal
 
 
+class PrincipalAddedToGroup(PASEvent):
+    implements(IPrincipalAddedToGroupEvent)
```

Comment:
Can you use the `@implementer` decorator? `implements` will not work in Python3

Reply:
Sounds sensible :)